lib/topological_inventory/mock_source/entity/volume.rb

```diff
@@ -7,8 +7,7 @@ def to_hash
         {
           :source_ref        => @uid,
           :name              => @name,
-          :uid_ems           => @uid,
-          :power_state       => power_states.sample,
+          :state             => power_states.sample,
```

Comment:
@slemrmartin Did you run into an error with this in testing? I'm interested in what led to this PR in the first place.

Reply:
yes, this was implemented a year ago, before that non-existing keys was just ignored, but it doesn't fit openapi specification (thus it's failing now)